features/step_definitions/barclamp_rabbitmq/rabbitmq_steps.rb

```diff
@@ -0,0 +1,79 @@
+Given(/^the barclamp proposal is using "rabbitmq" as messaging backend$/) do
+    json_response = JSON.parse(admin_node.exec!("crowbar rabbitmq show default").output)
+    expect(json_response["attributes"]["rabbitmq"]).not_to be_empty
+end
+
+When(/^the node with rabbitmq-server role has been detected successfully$/) do
+    json_response = JSON.parse(admin_node.exec!("crowbar rabbitmq show default").output)
+    @node_fqdn = json_response["deployment"]["rabbitmq"]["elements"]["rabbitmq-server"].first
+    expect(@node_fqdn).not_to be_empty
+end
+
+Then(/^I can check the status of rabbitmq-server using command line$/) do
+    rabbitmq_status = admin_node.exec!("rabbitmqctl status").output.lines[1..-1] 
+    rabbitmq_pid = 0
+    rabbitmq_status.each do |line|
+        line = line[0..-2]
+        scan_list = line.scan(/\{([^\}]*)\}/)[0][0].split(',')
+        if scan_list[0] == "pid"
+            rabbitmq_pid = scan_list[1].to_i
+            break
+        end
+    end
+    puts "rabbitmq serves with PID:", rabbitmq_pid
+    expect(@rabbitmq_pid).not_to equal(0)
+end
+
+And(/^I can add a new user and view the user in the users list$/) do
+    username = "rabbit"
```

Comment:
can you add that as a step global variable / constant instead of copy/pasting it in every step?